unix_integration/src/idprovider/kanidm.rs

```diff
@@ -201,6 +201,22 @@ impl IdProvider for KanidmProvider {
         }
     }
 
+    async fn unix_user_authenticate_continue(
+        &self,
+        id: &Id,
+        resp: &str,
+        data: Option<String>,
```

Comment:
If they're unused you have to mark them as such with underscores - build blocker.
```suggestion
        _id: &Id,
        _resp: &str,
        _data: Option<String>,
```